frontend/src/components/House/House.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import {css} from 'aphrodite';
+import S from './styles';
```

Comment:
I don't like the single letter variable names, make them descriptive, so we don't have to go looking around what it means, `styles` would probably work